ludwig/utils/trainer_utils.py

```diff
@@ -244,23 +256,49 @@ def log_metrics(self):
         return log_metrics
 
 
+def add_metrics_to_printed_table(
+    printed_table: List,
```

Comment:
```suggestion
    printed_table: Dict[str, List],
```
Also consider above suggestion to turn this into a class.